zipkin-server/src/main/java/zipkin2/server/internal/MetricsHealthController.java

```diff
@@ -85,7 +85,7 @@ public ObjectNode fetchMetricsFromMicrometer() {
   // Delegates the health endpoint from the Actuator to the root context path and can be deprecated
   // in future in favour of Actuator endpoints
   @Get("/health")
-  public HttpResponse getHealth() throws JsonProcessingException {
+  public AggregatedHttpResponse getHealth() throws JsonProcessingException {
```

Comment:
side q: will any of the other read methods end up blocking the event loop? ex getTraces etc?

Reply:
Nope they all return `AggregatedHttpResponse` so don't run on the event loop. Though I'm definitely interested in eventually migrating them to use `enqueue` instead of `execute` to be fully async :)